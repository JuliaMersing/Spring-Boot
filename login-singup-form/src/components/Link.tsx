import React from 'react';

type LinkProps = {
  children: React.ReactNode;
  className: string
  href: string,
}

const Link = ({
  children,
  className,
  href,
}: LinkProps) => (
  <a href={href} className={className}>{children}</a>
);

export default Link;
