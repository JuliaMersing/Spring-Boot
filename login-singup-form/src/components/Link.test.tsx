import React from 'react';
import { render, screen } from '@testing-library/react';
import Link from './Link';

describe('Link component', () => {
  test('should match with snapshot', () => {
    render(<Link href="/#" className="link">
      Forgot
      password?
    </Link>);
    screen.getByRole('link');
    expect('link')
      .toMatchSnapshot();
  });
});
