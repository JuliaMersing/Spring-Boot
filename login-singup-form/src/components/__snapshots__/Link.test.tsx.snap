// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component should match with snapshot 1`] = `"link"`;
